src/main/java/org/opensearch/search/relevance/transformer/kendraintelligentranking/pipeline/KendraRankingResponseProcessor.java

```diff
@@ -0,0 +1,174 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.search.relevance.transformer.kendraintelligentranking.pipeline;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.opensearch.action.search.SearchRequest;
+import org.opensearch.action.search.SearchResponse;
+import org.opensearch.action.search.SearchResponseSections;
+import org.opensearch.env.Environment;
+import org.opensearch.ingest.ConfigurationUtils;
+import org.opensearch.search.SearchHits;
+import org.opensearch.search.aggregations.InternalAggregations;
+import org.opensearch.search.internal.InternalSearchResponse;
+import org.opensearch.search.pipeline.Processor;
+import org.opensearch.search.pipeline.SearchResponseProcessor;
+import org.opensearch.search.profile.SearchProfileShardResults;
+import org.opensearch.search.relevance.transformer.kendraintelligentranking.KendraIntelligentRanker;
+import org.opensearch.search.relevance.transformer.kendraintelligentranking.client.KendraClientSettings;
+import org.opensearch.search.relevance.transformer.kendraintelligentranking.client.KendraHttpClient;
+import org.opensearch.search.relevance.transformer.kendraintelligentranking.configuration.KendraIntelligentRankingConfiguration;
+
+import static org.opensearch.search.relevance.transformer.kendraintelligentranking.configuration.Constants.KENDRA_DEFAULT_DOC_LIMIT;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * This is a {@link SearchResponseProcessor} that applies kendra intelligence ranking
+ */
+public class KendraRankingResponseProcessor implements SearchResponseProcessor {
+    /**
+     * key to reference this processor type from a search pipeline
+     */
+    public static final String TYPE = "kendra_ranking";
+    private final List<String> titleField;
+    private final List<String> bodyField;
+    private final int docLimit;
+    private final String tag;
+    private final String description;
+    private KendraHttpClient kendraClient;
+
+    private static final Logger logger = LogManager.getLogger(KendraRankingResponseProcessor.class);
+
+    /**
+     * Constructor that apply configuration for kendra re-ranking
+     *
+     * @param tag          processor tag
+     * @param description  processor description
+     * @param titleField   titleField applied to kendra re-ranking
+     * @param bodyField    bodyField applied to kendra re-ranking
+     * @param docLimit     docLimit applied to kendra re-ranking
```

Comment:
Looks like the build is complaining about the Javadoc for this param. Should be `@param inputDocLimit`.

Reply:
fixed docstring